src/plugins/intel_cpu/src/transformations/transformation_pipeline.cpp

```diff
@@ -455,6 +456,7 @@ void Transformations::PreLpt(const std::vector<ov::element::Type>& defaultPrecis
     CPU_DISABLE_PASS_COMMON(manager, ov::pass::ConvertTopK11ToTopK3);
     CPU_DISABLE_PASS_COMMON(manager, ov::pass::HSwishDecomposition);
     CPU_DISABLE_PASS_COMMON(manager, ov::pass::MatMulConstTransposesExtraction);
+    CPU_DISABLE_PASS_COMMON(manager, ov::pass::ScaledDotProductAttentionDecomposition);
```

Comment:
1,2 Done.
3, may we collect all perf data after related PR is merged? Because maybe SDPA op itself is no improvement or even some drop, it is the dependency of other optimization, the performance of SDPA+other opt may be considered as a whole. One model data:
![image](https://github.com/openvinotoolkit/openvino/assets/70364073/cb048d7d-60c9-4a29-9f54-8269f5365294)

Reply:
For LLMs I agree - final hieristics will depend on upcoming optimizations.
I am mostly speeaking about regular transformers like berts or VITs, StableDiffusion, etc. RoPE or states will not affect such models at all, Given that we have to make sure SDPA optimization will not introduce any regressions.

Reply:
Will do the test.

Reply:
Disable SDPA in static shape scenario to avoid potential regression.